### PR TITLE
TST: troubleshoot inconsistent xfails

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Comment by WillAyd** [comment]
Thanks for taking a look at these - great to get these working again. Took a quick look at a couple builds and noticed they weren't always the same failures. Could split this up into a few PRs if easier to troubleshoot

**Comment by jbrockmendel** [comment]
black and flake8 are OK with this diff, pep8speaks must have its own config or a version mismatch?

**Comment by jbrockmendel** [comment]
@WillAyd can I get a "booyah"?

**Comment by WillAyd** [comment]
booyah it is

File: pandas/tests/arithmetic/test_datetime64.py [hunk 1]
```diff
@@ -666,6 +666,7 @@ def test_comparison_tzawareness_compat_scalars(self, op, box_with_array):
     # Raising in __eq__ will fallback to NumPy, which warns, fails,
     # then re-raises the original exception. So we just need to ignore.
     @pytest.mark.filterwarnings("ignore:elementwise comp:DeprecationWarning")
+    @pytest.mark.filterwarnings("ignore:Converting timezone-aware:FutureWarning")
```
Comment (jreback) [inline pandas/tests/arithmetic/test_datetime64.py]: same question as on another PR, is this hiding something?

**Review by jreback — CHANGES_REQUESTED** [review]
question

**Comment by jreback** [comment]
any reason we shouldn't do this for 0.25.1?

File: pandas/tests/arithmetic/test_datetime64.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/arithmetic/test_datetime64.py]: AFAICT just noise in the logs

**Comment by jbrockmendel** [comment]
> any reason we shouldn't do this for 0.25.1?

Not that I'm aware of

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @jbrockmendel

**Pull request merged**